config/configmapprovider/expand_test.go

```diff
@@ -30,7 +30,9 @@ import (
 func TestBaseRetrieveFailsOnRetrieve(t *testing.T) {
 	retErr := errors.New("test error")
 	exp := NewExpand(&mockProvider{retrieveErr: retErr})
-	defer exp.Shutdown(context.Background())
+	defer func() {
```

Comment:
based on https://github.com/open-telemetry/opentelemetry-collector/pull/4462#discussion_r753363716 it's better to use `t.Cleanup` instead

Reply:
done